process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -289,15 +289,15 @@
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Keep a copy of the persistent domain configuration even if this host is not the Domain Controller")
+    @Message(id = Message.NONE, value = "Keep a complete copy of the persistent domain configuration even if this host is not the Domain Controller. This parameter will override any ignore-unused-configuration setting in host.xml and attempt to store the complete domain configuration.")
```

Comment:
Yep, you're correct, --backup only sets it if there is no value set, otherwise whatever is set is used.

Do you think we should warn if this is set like this? (I think we should). It seems like something a user could potentially do and not realize, then not be able to promote the backup to DC.